components/next_steps_view/steps.ts

```diff
@@ -194,3 +196,15 @@ export const nextStepsNotFinished = createSelector(
         return !Steps.every(checkPref);
     },
 );
+
+export const userExistedForSomeTime = createSelector(
+    'userExistedForSomeTime',
```

Comment:
Naming here is a big vague, could we maybe switch it up to check if current user is a new account so it's more clear the intention of this - maybe something like: `isCurrentUserNewAccount` or something similar.